quora-service/src/main/java/com/upgrad/quora/service/dao/UserDao.java

```diff
@@ -76,4 +89,8 @@ public UserAuthTokenEntity createAuthToken(final UserAuthTokenEntity userAuthTok
     public void updateUser(final UserEntity updatedUserEntity) {
         entityManager.merge(updatedUserEntity);
     }
+
+    public void deleteAuthToken(final UserAuthTokenEntity deleteAuthToken){
```

Comment:
we need to update auth token then y updating user the parameter should be UserAuthEntity type?

Reply:
Done.